packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuideButton/FieldGuideButton.js

```diff
@@ -70,7 +69,6 @@ function FieldGuideButton (props) {
       theme={theme}
     />
   )
-
 }
 
```

Comment:
The aria-label was not used. Was this intended? Can we add the `aria-label` to `<StyledButton />`?

`aria-label={t('FieldGuide.FieldGuideButton.ariaLabel')}`

NOTE: must add
`const { t } = useTranslation('components')`
inside the `FieldGuideButton` function as well.

Reply:
The field guide button already has a text label, 'Field Guide'. `aria-label` is only needed when there's no text on a button eg. for icon buttons. Otherwise, I think you would end up with a button that has two duplicated labels.

Reply:
> @goplayoutside3 What is this file for? [plugins.json](https://github.com/zooniverse/front-end-monorepo/blob/classifier-fieldguide-i18next/packages/lib-classifier/src/translations/en/plugins.json)

The translation keys for components inside `/src/plugins` will eventually be organized into that file. I've separated those keys from everything in `/src/components` because we expect more tools to be added as plugins in the future and more keys added along the way.

Reply:
OK. If we're not going to use the `aria-label`, can we remove it from the dictionary ([components.json](https://github.com/zooniverse/front-end-monorepo/blob/classifier-fieldguide-i18next/packages/lib-classifier/src/translations/en/components.json))?

Reply:
> 

I'll remove the unused aria label key from the dictionary.